services/greasyfork/greasyfork-license.tester.js

```diff
@@ -0,0 +1,16 @@
+import { ServiceTester } from '../tester.js'
+
+export const t = new ServiceTester({
+  id: 'GreasyForkLicense',
+  title: 'Greasy Fork License',
+  pathPrefix: '/greasyfork',
+})
```

Comment:
This should use the helper model too, similar to comment above in previous tester